src/proteus/star/wrapper.py

```diff
@@ -130,7 +130,7 @@ def get_new_spectrum(t_star:float, config:Config,
     # Dummy case
     if config.star.module == 'dummy':
         from proteus.star.dummy import generate_spectrum
-        wl, fl = generate_spectrum(config.star.dummy.Teff, config.star.dummy.radius)
+        wl, fl = generate_spectrum(config.star.dummy.Teff, R_star)
```

Comment:
I removed passing R_star as an argument in my last PR as it is a constant for the dummy module... but I missed the unit conversion.

I recommand not passing R_star as an argument but rather replace l133 with
`wl, fl = generate_spectrum(config.star.dummy.Teff, config.star.dummy.radius * Rsun)`